### PR TITLE
Add iCal: option URL for ics() as URL;VALUE=URI: to display links in …

### PR DESCRIPTION
**Pull request by gulios**
…Calendar macOS

### CODE REVIEW

File: src/Generators/Ics.php [hunk 1]
```diff
@@ -31,6 +31,7 @@ public function generate(Link $link): string
             'BEGIN:VEVENT',
             'UID:'.($this->options['UID'] ?? $this->generateEventUid($link)),
             'SUMMARY:'.$this->escapeString($link->title),
+            'URL;VALUE=URI:'.($this->options['URL'] ?? ''),
```
Comment (lptn) [inline src/Generators/Ics.php:34]: We should not include URL if it's empty

**Review by lptn — CHANGES_REQUESTED** [review]
Thanks for the change!

File: src/Generators/Ics.php (hunk 1 above)
Comment (gulios) [reply src/Generators/Ics.php]: Ok, pls check it now

**Review by lptn — APPROVED** [review]
thanks a lot! 👍

**Pull request merged**